packages/ui-views/organisms/Header/header.tsx

```diff
@@ -16,31 +18,38 @@ interface UserProps {
 }
 
 const Header = CurrentUser(({ user, loading, error, theme }: UserProps) => {
-    let currentTheme = themeable(theme)
+    let currentTheme = themeable(theme);
     return (
-        loading ? <strong>Loading...</strong> : (error ? <p style={{ color: '#F00' }}>API error</p> : (
-        <header {...currentTheme(0, 'main_header')} >
-          <div {...currentTheme(1, 'row')}>
-            <Link href='/'><span  {...currentTheme(2, 'header_logo')} /></Link>
-            <div {...currentTheme(4, 'header_search')} >
-              <input {...currentTheme(5, 'search', 'input')} placeholder='Search on kamasi' />
-              <span {...currentTheme(6, 'search_icon')}><Search /></span>
+        loading ? <strong>Loading...</strong> : (error ? <p style={{ color: "#F00" }}>API error</p> : (
+        <header {...currentTheme(0, "main_header")} >
+          <div {...currentTheme(1, "row")}>
+            <Link href="/"><span  {...currentTheme(2, "header_logo")} /></Link>
+            <div {...currentTheme(4, "header_search")} >
+              <input {...currentTheme(5, "search", "input")} placeholder="Search" />
+              <span {...currentTheme(6, "search_icon")}><Search /></span>
             </div>
-            <div {...currentTheme(3, 'header_menu')} >
-                  <div {...currentTheme(17, 'menu_bell')}>
+            <div {...currentTheme(3, "header_menu")} >
+                  <div {...currentTheme(17, "menu_bell")}>
                     <Bell />
                   </div>
-                  <div {...currentTheme(14, 'menu_profile')} >
-                    <div {...currentTheme(15, 'profile_image')}>
+                  <div {...currentTheme(14, "menu_profile")} >
+                    <div {...currentTheme(15, "profile_image")}>
                       <img src={user.image} />
                     </div>
-                    <h4>{user.name || 'nobody'}</h4>
-                    <span {...currentTheme(16, 'profile_other')}><Horizontal /></span>
+                    <h4>{user.name || "nobody"}</h4>
+                    <span {...currentTheme(16, "profile_other")}>
+                      <div {...currentTheme(17, "dropdown")}>
+                        <span>°°°</span>
+                        <div {...currentTheme(18, "dropdown-content")}>
+                          <Link to={"/"} {...currentTheme(19, "link")}>home</Link>
```

Comment:
It might be helpful to add a comment here letting future devs know this is where you add more paths to the dropdown. The structure for this file is pretty long and dense, so that might save them some time.

Reply:
That sounds like a good Idea